src/includes/class-wordlift-website-jsonld-converter.php

```diff
@@ -98,8 +108,8 @@ private function set_search_action( &$params ) {
 		 */
 		$search_url = apply_filters( 'wl_jsonld_search_url', home_url( '/' ) . '?s={search_term_string}' );
 
-		// Add search action
-		$params['potentialAction'] = array(
+		// Add search action to main item in JSON-LD.
+		$jsonld[0]['potentialAction'] = array(
```

Comment:
Have the caller send already the index 0, so that we don't spread responsibilities - even though we're in the same class.